indra/newview/llviewershadermgr.cpp

```diff
@@ -331,6 +331,7 @@ void LLViewerShaderMgr::finalizeShaderList()
     mShaderList.push_back(&gDeferredPBROpaqueProgram);
     mShaderList.push_back(&gDeferredAvatarProgram);
     mShaderList.push_back(&gDeferredTerrainProgram);
+    mShaderList.push_back(&gDeferredPBRTerrainProgram);
```

Comment:
This one's interesting. Mirror uniforms are applied in `LLEnvironment::update`, but the PBR terrain shader wasn't in the list of shaders, so `mUniformsDirty` was never set to true. I am sure mirrors weren't the only thing affected by this.